### ISSUE TITLE
Package keywords suggestions

### ISSUE THREAD

**Issue by mk-pmb** [papandreou/node-mountfs#4]
Hi, thanks for this module! Looks like I don't have to write it myself, after all. However, it was by sheer luck that I found it, since npm didn't find it for my searches like "mount fs", "fs mount", "virtual fs" and so on. May I suggest you add these?

``` js
  "keywords": [
    "file system",
    "filesystem",
    "fs",
    "mount point",
    "mount points",
    "mount",
    "mountpoint",
    "mountpoints",
    ],
```

("fs" included for easy copying.)
Also, what's the git aspect that made you add "git" as a keyword?

**Comment by gustavnikolaj** [comment]
I'm glad that you found it any way, and I'll update it immediately.

It looks like it's just a copy/paste thing from one of a few related modules:
- https://github.com/papandreou/node-fsplusgit/blob/master/package.json#L24-L27
- https://github.com/papandreou/node-gitfakefs/blob/master/package.json#L22-L25

It doesn't make a lot of sense with the git keyword in this module :-)

**Comment by gustavnikolaj** [comment]
released with version 0.4.2 :-)

**Comment by mk-pmb** [comment]
Thanks! :+1:
